rpm/grass.spec

```diff
@@ -37,64 +37,66 @@ BuildRequires:	flexiblas-devel
 %else
 BuildRequires:	blas-devel, lapack-devel
 %endif
+BuildRequires:	bzip2-devel
 BuildRequires:	cairo-devel
-BuildRequires:	gcc-c++
 BuildRequires:	desktop-file-utils
 BuildRequires:	fftw-devel
 BuildRequires:	flex
 BuildRequires:	freetype-devel
+BuildRequires:	gcc-c++
 BuildRequires:	gdal-devel
 BuildRequires:	geos-devel
 BuildRequires:	gettext
 BuildRequires:	laszip-devel
 BuildRequires:	libappstream-glib
 BuildRequires:	libpng-devel
+%if 0%{?rhel} && 0%{?rhel} == 7
+BuildRequires:	postgresql-devel
+%else
+BuildRequires:	libpq-devel
+%endif
 BuildRequires:	libtiff-devel
 BuildRequires:	libXmu-devel
-BuildRequires:	mesa-libGL-devel
-BuildRequires:	mesa-libGLU-devel
+BuildRequires:	libzstd-devel
+BuildRequires:	make
 %if (0%{?rhel} > 7 || 0%{?fedora})
 BuildRequires:	mariadb-connector-c-devel openssl-devel
 %else
 BuildRequires:	mysql-devel
 %endif
+BuildRequires:	mesa-libGL-devel
+BuildRequires:	mesa-libGLU-devel
 BuildRequires:	netcdf-devel
+BuildRequires:	PDAL
+BuildRequires:	PDAL-devel
+BuildRequires:	PDAL-libs
+BuildRequires:	proj-devel
 BuildRequires:	python3
 %if 0%{?rhel} == 7
 # EPEL7
-BuildRequires:	python%{python3_version_nodots}-numpy
-%else
-BuildRequires:	python3-numpy
-%endif
-%if 0%{?rhel} && 0%{?rhel} == 7
-BuildRequires:	postgresql-devel
+BuildRequires:	python%{python3_version_nodots}-dateutil
 %else
-BuildRequires:	libpq-devel
+BuildRequires:	python3-dateutil
```

Comment:
I am not sure about the changes in this SPEC file, also in the lines above and below.

Note that the upstream file is here (once settled here I can fwd-port desired changes):
https://src.fedoraproject.org/rpms/grass/blob/rawhide/f/grass.spec

Reply:
If the order of the dependencies (for a given section) doesn't matter, it is equivalent and no lines are lost, and the same is in the if else blocks. If the order matters for example on their side knowing what package has to be built before (at an ecosystem level), then we might need to revert the changes here.

Reply:
Right. I have extracted `grass.spec` from this PR and built it locally with Fedora tools, all fine.
Hence updated accordingly https://src.fedoraproject.org/rpms/grass/blob/rawhide/f/grass.spec

My suggestion would be to merge that SPEC file back into this PR.

Reply:
I merged back the spec file, keeping the 8.4.0 release 3 version, as updating it to 8.5 would probably need a changelog entry, and isn't synced back to rawhide.

Reply:
Great, thanks a lot.